.travis.yml

```diff
@@ -1,11 +1,7 @@
 language: objective-c
-before_install:
-  - gem install cocoapods
+osx_image: xcode10.2
 before_script:
-  - brew update
-  - brew upgrade xctool && brew cleanup xctool
-  - xctool --version
```

Comment:
Why did you remove these steps? What happens if cocoapods is not installed  for instance?

Reply:
I don't remember the details, but it wasn't the case when I created the project, 5 years ago.
And xctool wasn't used anymore, so no problem with that.